docs/pt-br/contribute/core.md

```diff
@@ -56,15 +56,15 @@ bats test/list_commands.bash
 
 ::: tip
 
-**Add tests!** - Tests are **required** for new features and speed up review of bug fixes. Please cover new code paths before you create a Pull Request. See [bats-core documentation](https://bats-core.readthedocs.io/en/stable/index.html)
+ **Adicione testes!** - Os testes são **necessários** para novos recursos e aceleram a revisão de correções de bugs.  Por favor, cubra novos caminhos de código antes de criar um Pull Request.  Consulte [documentação do bats-core](https://bats-core.readthedocs.io/en/stable/index.html)
 
 :::
 
-## Bats Testing
```

Comment:
Bats is an acronym that stands for Bash Automated Testing System, so you may want to leave this as `BATS` in the translated text.

Reply:
If you want to create another PR to address this that would be great. I went ahead and merged your PR.